lib/src/cli/cli.dart

```diff
@@ -140,9 +140,10 @@ const _ignoredDirectories = {
   '.fvm',
 };
 
-bool _isPubspec(FileSystemEntity entity) {
+bool _isPubspec(FileSystemEntity entity, Set<String> ignore) {
```

Comment:
I don't think we should pass the ignore list into this function since this function should be specifically determining whether a `FileSystemEntity` is a pubspec.

Reply:
What do you think of having bool checks like this:
```dart
bool _isPubspec(FileSystemEntity entity) {
  if (entity is! File) return false;
  return p.basename(entity.path) == 'pubspec.yaml';
}

bool _isIgnored(FileSystemEntity entity, Set<String> ignore) {
  final segments = p.split(entity.path).toSet();
  if (segments.intersection(_ignoredDirectories).isNotEmpty) return true;
  if (segments.intersection(ignore).isNotEmpty) return true;
  return false;
}
```

and ran with:
```dart
final processes = _Cmd.runWhere<T>(
  run: (entity) => cmd(entity.parent.path),
  where: (entity) => !_isIgnored(entity, ignore) && _isPubspec(entity),
  cwd: cwd,
);
```

Reply:
Yeah that looks better to me. We could even add a simple extension method:

```dart
final processes = _Cmd.runWhere<T>(
  run: (entity) => cmd(entity.parent.path),
  where: (entity) => ignore.excludes(entity) && _isPubspec(entity),
  cwd: cwd,
);
```